testing/tests/DevExpress.ui.widgets/list.markup.tests.js

```diff
@@ -314,6 +314,75 @@ if(devices.real().deviceType === 'desktop') {
     });
 }
 
+QUnit.module('Accessibility', ()=> {
```

Comment:
I think that accessibility module should include checking of aria attributes. Let's name it 'Select by click' by functionality that we check.

Reply:
Or let's move this tests to 'selection module' in list/commonTests.js

Reply:
Fixed